test/e2e/e2e_test.go

```diff
@@ -1720,6 +1720,16 @@ func TestE2E(t *testing.T) {
 				},
 			},
 		}
+		// webhooks to add after initial creation to ensure
+		// updates can be made for more than the original number of webhooks.
+		webhooksToAdd := []admissionreg.MutatingWebhook{
```

Comment:
These are e2e-tests. While those are fine, do you think you could add unit tests for the changed code as well?

Reply:
I can, but will take me a bit longer to get back to that since there doesn't appear to be any existing wire up for kubeclient's there

Reply:
Added some tests, but not sure how correct they are. Also posted a question [here](https://github.com/openshift/service-ca-operator/pull/207/files#r1173145338)